src/sas/qtgui/Perspectives/Invariant/InvariantDetails.py

```diff
@@ -77,13 +78,20 @@ def showDialog(self):
         # Pull out data from the model
         self.qstar_total = float(self._model.item(WIDGETS.W_INVARIANT).text())
 
-        self.txtQData.setText(str(self.qstar_total))
-        self.txtQDataErr.setText(self._model.item(WIDGETS.W_INVARIANT_ERR).text())
-
         # Reset progress counters
         self.progress_low_qstar = 0.0
         self.progress_high_qstar = 0.0
-        self.progress_qstar = 100.0
+        self.progress_data_qstar = 100.0
+
+        # Q* from data
+        self.qdata = float(self._model.item(WIDGETS.D_DATA_QSTAR).text())
```

Comment:
This will fail with `ValueError` when the Q* calculations fail leaving the field `D_DATA_QSTAR` empty. Would be good to surround it with `try..except`

Reply:
Hummm... so I just copied this from the low q and hi q code, neither of which uses a try except.  Should there be a try except on all three?

Reply:
OK. These fields can be left without `try...except` but should probably be made read-only. Currently they can be modified but modifications don't do anything since these values are output only. This is more of a 'would be nice and less confusing to the user' than 'it will wreak havoc if users enter junk' category, hence can safely be taken out of the PR and made a separate issue to address in 5.0.4/5.1